index.html

```diff
@@ -174,6 +174,24 @@ <h2 class="connect">LET'S CONNECT</h2>
             </div>
         </div>
     </section>
+    <section>
+        <form action="https://formspree.io/f/mqkjgqke" method="post">
+        <div class="for-con">
+        <h1>Contact me</h1>
+        <h4>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </h4>
```

Comment:
- [OPTIONAL] I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).